ui/app/src/models/Site.ts

```diff
@@ -107,13 +107,15 @@ export interface Site {
   name: string;
   description: string;
   imageUrl: string;
+  state?: 'INITIALIZING' | 'READY' | 'DELETING' | 'DELETED';
```

Comment:
Why optional?

Reply:
That interface is also used before creating a Site, and at that moment, there is no state